spec/bowling_SC_spec.js

```diff
@@ -0,0 +1,68 @@
+
+describe('bowlingGame', function() {
+
+  beforeEach(function() {
+    game = new bowlingGame();
+  });
+
+  it('can create new game', function() {
+    let game = new bowlingGame();
+    expect(game).toBeInstanceOf(bowlingGame);
+  });
+
+  it('can roll a gutter game', function() {
+    for (let i = 0; i < 20; i++) {
+      game.firstRoll(0);
```

Comment:
How come we don't also use `secondRoll` here? Unless I'm misunderstanding what the functions are meant to be used for, as a user I would expect to alternate between calling `firstRoll` and `secondRoll` so we should mimic how a user would use our application in the tests